### ISSUE TITLE
View constructor in main

### ISSUE THREAD

**Issue by jackonelli** [jaklindq/RicochetClone#4]
Robotubbies.java has an outdated view constructor! Need additional parameter: event listener

**Comment by JonathanFager** [comment]
Does it not make more sense that view itself creates this?

**Comment by Niklas4** [comment]
Nope, controller needs to have this and send it with the constructor since View should mainly paint components and not handle logic
